src/collection.ts

```diff
@@ -537,30 +537,30 @@ export abstract class BaseCollection implements C8Collection {
     );
   }
 
-  createFulltextIndex(fields: string[] | string, minLength?: number) {
+  createFulltextIndex(fields: string[] | string, opts?: any) {
```

Comment:
I know we have been supporting `any` for object shapes at multiple places. But if we know the object shape for this one, we should be more concrete. Otherwise use something similar to`Record<string, any>` atleast.